README.md

```diff
@@ -548,6 +550,34 @@ To deactivate the `gitsome` `virtualenv`, run:
 
     $ deactivate
 
+### Running as Docker Container
```

Comment:
Should this be "Running as a Docker Container"?  If we make this change, please update line 180